cli/command/container/completion.go

```diff
@@ -122,6 +122,7 @@ func addCompletions(cmd *cobra.Command, dockerCli command.Cli) {
 	_ = cmd.RegisterFlagCompletionFunc("publish", completion.NoComplete)
 	_ = cmd.RegisterFlagCompletionFunc("pull", completion.FromList(PullImageAlways, PullImageMissing, PullImageNever))
 	_ = cmd.RegisterFlagCompletionFunc("restart", completeRestartPolicies)
+	_ = cmd.RegisterFlagCompletionFunc("runtime", completion.NoComplete)
```

Comment:
We _could_ possibly provide some amount of completion for this (known runtimes in the daemon), but not sure if we should, given that with containerd as runtime, alternative runtimes can be used that are not defined in the daemon config.